components/organisms/LearningPath/TypeResource.tsx

```diff
@@ -7,8 +8,22 @@ export const TypeResource = () => {
   const selectedResourceTopic = useAppSelector(
     (state) => state.resource.selectedResource
   );
-  const resourceType = selectedResourceTopic?._id.typeResource;
- 
+
+  console.log("Selected Resource Topic:", selectedResourceTopic);
```

Comment:
I'm not sure if we need this logs in the user console...

Reply:
delete console